Sender/AboutBMSSender.md

```diff
@@ -0,0 +1,31 @@
+# BMS Sender
+
+This document is containing the details about BMS Sender
+
+## BMS Sender Classes
+
+Two main classes are:
+
+### BatteryChargingParametersGenerator:
+
+This class is responsible for generating the random values for two battery charging parameters which are Temperature and State of Charge.
+
+### BatteryChargingParametersStreamer: 
+
+This class displays the two battery charging parameters (Temperature and State of Charge) in console continuously, with a delay of 2000 milliseconds, until the user presses escape (ESC) key from the keyboard.
+
+## Sample of Console Output 
+
+Battery Charging Parameters (Press Escape to exit)
```

Comment:
Thoughtful documentation - this will really help the implementer of the receiver.
Suggestion to also build tests to check this output format. Without these tests, a future maintainer may change the output format. It will get noticed only when we integrate with the receiver. That would take way more time than a local test.